main.py

```diff
@@ -57,25 +56,26 @@ def init(config: confuse.core.Configuration):
     logging.info(f"Starting Chiadog ({version()})")
 
     # Create log consumer based on provided configuration
-    chia_logs_config = config['chia_logs']
-    log_consumer = create_log_consumer_from_config(chia_logs_config)
+    chia_logs_config = config["chia_logs"]
+    log_consumer: LogConsumer = create_log_consumer_from_config(chia_logs_config)
     if log_consumer is None:
         exit(0)
```

Comment:
nit: this check isn’t needed anymore.

Reply:
Fixed.

Reply:
ah, here I actually meant that we don’t need the if-statement since the function doesn’t anymore return `None`

Reply:
But its a nit, happy to merge as-is